backend/routes/stories.js

```diff
@@ -0,0 +1,41 @@
+const express = require("express");
+const router = express.Router();
+const db = require("../config/database");
+
+const Story = db.Story;
+
+router.get("/", (req, res, next) => {
+    let page = req.query.page || 0
+    let limit = 50;
+    Story.findAll({
+        offset: limit * page,
+        limit: limit,
+        order: [['id', 'DESC']]
+    }).then(data=>res.send(data))
+
+});
+router.get("/getAll", (req, res, next) => {
+    Story.findAll({
+    }).then(data=>res.send(data))
+
+});
+router.get("/:id", (req, res, next) => {
+    Story.findAll({ where: {
```

Comment:
Shouldn't this `Story.findAll()` be a `findOne()` or something?

Reply:
fixed